lightning/src/ln/onion_route_tests.rs

```diff
@@ -14,51 +12,79 @@
 //! returned errors decode to the correct thing.
 
 use crate::chain::channelmonitor::{CLTV_CLAIM_BUFFER, LATENCY_GRACE_PERIOD_BLOCKS};
-use crate::sign::{EntropySource, NodeSigner, Recipient};
 use crate::events::{Event, HTLCHandlingFailureType, PathFailure, PaymentFailureReason};
-use crate::types::payment::{PaymentHash, PaymentSecret};
 use crate::ln::channel::EXPIRE_PREV_CONFIG_TICKS;
-use crate::ln::channelmanager::{HTLCForwardInfo, FailureCode, CLTV_FAR_FAR_AWAY, DISABLE_GOSSIP_TICKS, MIN_CLTV_EXPIRY_DELTA, PendingAddHTLCInfo, PendingHTLCInfo, PendingHTLCRouting, PaymentId, RecipientOnionFields};
-use crate::ln::onion_utils::{self, LocalHTLCFailureReason};
-use crate::routing::gossip::{NetworkUpdate, RoutingFees};
-use crate::routing::router::{get_route, PaymentParameters, Route, RouteParameters, RouteHint, RouteHintHop, Path, TrampolineHop, BlindedTail, RouteHop};
-use crate::types::features::{InitFeatures, Bolt11InvoiceFeatures};
+use crate::ln::channelmanager::{
```

Comment:
@TheBlueMatt I am fine with the file as is, but let me know what formatting you think should be updated at a minimum.